File: app/src/main/java/com/example/admin/smarttranslator/Fragments/CoffeeFragment.java
package com.example.admin.smarttranslator.Fragments;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.admin.smarttranslator.R;

public class CoffeeFragment extends Fragment {

    public static CoffeeFragment newInstance() {
        return new CoffeeFragment();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.coffee_layout, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {

    }
}

File: app/src/main/java/com/example/admin/smarttranslator/Fragments/CoffeeFragment.kt
package com.example.admin.smarttranslator.Fragments

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.example.admin.smarttranslator.R

class CoffeeFragment : androidx.fragment.app.Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.coffee_layout, container, false)
    }

    override fun onViewCreated(@NonNull view: View, @Nullable savedInstanceState: Bundle?) {

    }

    companion object {

        fun newInstance(): CoffeeFragment {
            return CoffeeFragment()
        }
    }
}
